MobileSecondPage.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&family=Poppins:ital,wght@0,200;0,500;0,600;0,800;1,100&display=swap" rel="stylesheet">
+    <title>runner world cup</title>
+    <link rel="stylesheet" href="style_2.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"
+      integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+</head>
+<body>
+    <span class="upperhead">
+        <div>
+            <img src="./images/Rectangle.png" alt="">
+            <img src="./images/twitter.png" alt="">
+            <a href="#">English</a>
+            <a href="#">MyPage</a>
+            <a href="#">Logout</a>
+        </div>
+    </span>
+    <header>
+        <nav>
+            <div class="burger">
+                <img src="./images/mobile_menu 1.png" alt="" />
+            </div>
+                <div class="header-menu">
+                <span class="close">X</span>
+                <a class="links" id="about" href="./MobileSecondPage.html">About</a>
+                <a class="links" id="myProgram" href="#myProgram">Program</a>
+                <a class="links" id="join" href="#join">Join</a>
+                <a class="links" id="sponsor" href="#sponsor">Sponsor</a>
+                <a class="links" id="news" href="#news">news</a>
+                <a href="#" id="DeskIcon">Worldcup</a>
+            </div>
+        </nav>
```

Comment:
Good looking navbar ⛵ it really helps to know where you can go on the page, the mobile version design is almost perfect but it lacks something, the navbar background should be transparent or the same color as the next section (the header) since the header has a background image, I'd suggest you go for the transparency. 🤔